docs/Initial-Setup/Setting-up-your-Manager-Instance.rst

```diff
@@ -20,44 +26,50 @@ right corner, ensure that the correct region is selected.
 
 To launch a manager instance, follow these steps:
 
-1. From the main page of the EC2 Management Console, click
-   ``Launch Instance``. We use an on-demand instance here, so that your
+#. From the main page of the EC2 Management Console, click
+   *Launch Instance*. We use an on-demand instance here, so that your
    data is preserved when you stop/start the instance, and your data is
    not lost when pricing spikes on the spot market.
```

Comment:
```suggestion
   not lost when pricing spikes on the spot market.
#. In the *Name* field, give the instance a recognizable name, for example ``firesim-manager-1``. This is purely for your own convenience and can also be left blank.
```

Reply:
(pointless, but not telling users what to put for the first field on the page felt wrong)